custom_components/fellow/fellow_aiden/__init__.py

```
@@ -194,287 +245,312 @@ def __device(self):
+        response = await self._request_with_reauth(
+            "post", schedule_url, json=data
+        )
+        await self._ensure_success(response, "Schedule creation")
+
+        parsed = await self._parse_response(response)
+        if "id" not in parsed:
+            message = parsed.get("message", "Unable to get error message.")
+            if "Profile could not be found" in message:
+                ids = await self._get_profile_ids()
+                message += f"Valid profiles: {ids}"
+            raise Exception(f"Error in processing: {message}")
```

Ruff (0.15.2)

[warning] Create your own exception

(TRY002)